tools/rs-server/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+#  minimum required cmake version: 3.1.0
+cmake_minimum_required(VERSION 3.9.0)
+
+project(rs-server VERSION 1.0.0 LANGUAGES CXX C)
+
+# Save the command line compile commands in the build output
+set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
+
+set(DEPENDENCIES ${DEPENDENCIES} realsense2)
+
+set(CMAKE_CXX_FLAGS "-DNEWLOCALE_NOT_USED=1 -DBSD=1 -DSOCKLEN_T=socklen_t -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE=1 -DALLOW_RTSP_SERVER_PORT_REUSE=1 -DNO_OPENSSL=1")
```

Comment:
It doesn't make sense to override all librealsense CXX flags 
In particular, I had to add `-latomic` to build on the RPi

Reply:
Fixed